packages/odata-v4/src/filter-functions.ts

```diff
@@ -152,7 +152,7 @@ export function hasSubsequence<
   return filterFunction('hassubsequence', 'boolean', subsequence, sequence);
 }
 /**
- * TODO: totalseconds, time
+ * TODO: totalseconds, time.
```

Comment:
General question: Is it a manual effort to add the "."?

Reply:
No, the jsdoc rule `jsdoc/require-description-complete-sentence` does this automatically. It was added in the other PR #2214 . I fixed it in a separate PR so its easier to review

Reply:
I guess you enabled this rule then run the lint fix.
The rule was removed for making the pipeline happy in the previous PR, right?
Do we plan to add the rule in the future when everything is fixed?